### ISSUE TITLE
[Item] Warrior T10 4P bug

### ISSUE THREAD

**Issue by plznoz** [dalaranwow/dalaran-wow#4798]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If you get 2 slam procs close enough together, and the 2nd of them is a 4p proc (2 stacks), none of them will have a reduced cooldown (0.5 sec from the 4P).

**Expected behaviour**: Tell us what should happen instead
It should still be reduced even if the slam procs are close.

**Steps to reproduce the problem**:

1. Get a slam proc and use it
2. Use Bloodthirst (repeat 1 and 2 until you get a double stack proc as the 2nd proc)
3. Use 2 slams in a row, observe the gcd not being reduced on both

TLDR: Slam > BT > Slam > Slam

**Include proofs for this behaviour**
2 Examples from Dalaran today:
https://youtu.be/qNM3KgOaVB8
https://youtu.be/KnX7j11xHvc

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Slam! (the proc)
46916

Slam (the spell)
50783

Item - Warrior T10 Melee 4P Bonus
70847

**Comment by hairybolz** [comment]
wow.... this just doesn't work at all does it...

**Comment by TeridanDW** [comment]
Hello @plznoz, thank you for the report.
Could you confirm or deny if this is still happening on live realms, because I can't reproduce it on ptr (following your steps)?

**Comment by plznoz** [comment]
I don't play the server any more, but since the last class fixes (afaik) was back in Ulduar I don't see how it could have been fixed.

**Comment by AttendantTock** [comment]
Hello,

I have managed to reproduce the issue: https://youtu.be/z_v7PogiJ5M

Recap of the video:
1 - Use whirlwind
2 - Get double Slam! proc
3 - Use both proc, see the reduced GCD
4 - Use Bloodthirst
5 - Get double Slam! proc
6 - Use both proc, see the ***not*** reduced GCD

**Comment by hrtyv** [comment]
I can also confirm this issue, the first double proc you get will always be with reduced GCD and if you get another one after with bloodthirst the GCD is not reduced
